.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Resolves #ISSUE.
+
+**No (!) PRs should be merged without at least one review from a different team member of the Mondo Technical team.**
+
+## Documentation
+
+- [ ] `docs/` have been added/updated **OR**
+- [ ] No updates to the docs is considered unnecessary after careful consideration.
+
+## QC
+
+- [ ] `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
+- [ ] No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary.
+
+## High-level description
+
```

Comment:
A couple other misc PR reminders in my notes. DK if you want to add / not add these, and if so, do as just a markdown comment for optional guidance, or add a checkbox.
- "3-word commit headers"
- Any new or upgraded packages? --> Open an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) to request addition. Can also open a PR. Python package changes should occur in [[requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full)](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full), and non-Python packages in [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile).

Reply:
> By the way, is this SOP that we are building for more than just mondo-ingest?

I would suggest for now, just mondo-ingest. Later we can spread it more around.